apis/storage/v1alpha1/referencers.go

```diff
@@ -46,12 +48,12 @@ func (mg *S3BucketPolicy) ResolveReferences(ctx context.Context, c client.Reader
 		Extract:      reference.ExternalName(),
 	})
 	if err != nil {
-		return err
+		return errors.Wrap(err, "spec.forProvider.bucketName")
```

Comment:
I think this needs a rebase. S3 v1beta1 is merged and bucket name is now in external-name annotation.

Reply:
I think this one is working as expected (it's for the bucket policy), but I've rebased and updated the bucket ones too.